### PR TITLE
Defining a function named renderEntry with a single parameter named e…

### PR DESCRIPTION
**Pull request by Aiden-Peace300**

### CODE REVIEW

**Comment by Aiden-Peace300** [comment]
user can view their entry REVIEW

File: index.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
         <div class="column-full">
           <div class="row">
             <h1 id="header-style">Code Journal</h1>
+            <a id="navbar-entries-link" href="#entries"></a>
```
Comment (shawnkost) [inline index.html]: There should be some text inside this `a` element. It should say "Entries"

File: index.html [hunk 2]
```diff
@@ -88,7 +89,23 @@ <h2>New Entry</h2>
...
+              <div class="new new-button-left-of-column-full">
+                <a class="new-button" href="#entry-form" id="new-entry-button"
+                  >New</a
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+        <ul id="entries-list">
+          <li class="no-entries-message">No entries have been recorded yet.</li>
```
Comment (shawnkost) [inline index.html]: This shouldn't be inside the `ul`. This should just be a `p` element outside the `ul`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple tweaks here Aiden!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**